pontoon/static/js/widgets/errors.js

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+
+
+export default class ErrorList extends React.Component {
+
+    render () {
+        const {errors} = this.props;
+        if (!Object.keys(errors).length > 0) {
```

Comment:
This is confusing. Why not just `length === 0`? Or `!length`?

Reply:
ill update